src/pages/Calendar/FullCalendar.tsx

```diff
@@ -18,28 +19,55 @@ const CalendarPage = () => {
   ]);
 
   const calendarRef = useRef<any>(null);
```

Comment:
```suggestion
  const calendarRef = useRef<FullCalendar>(null);
```
타입을 명확히 지정해주면 더 좋겠죠!

Reply:
넵! Fullcalendar로 변경하면서
undefined 일 때를 고려해서 아래와 같이 수정하였습니다.
```
const calendarApi = calendarRef.current?.getApi();
if (!calendarApi) return;
```